app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/edit/serializer.phtml

```diff
@@ -4,12 +4,15 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Ajax\Serializer */
+/**
+ * @var $block \Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Ajax\Serializer
+ * @var \Magento\Framework\Escaper $escaper
+ */
 ?>
 
 // phpcs:disable Magento2.Security.InsecureFunction.DiscouragedWithAlternative
```

Comment:
Could you wrap it to the php tag in order not to print it?
Also check - it might be not needed